src/setupWorker/start/utils/printStartMessage.ts

```diff
@@ -1,14 +1,20 @@
+import { SetupWorkerInternalContext } from '../../glossary'
 import { devUtils } from '../../../utils/internal/devUtils'
 
 interface PrintStartMessageArgs {
   quiet?: boolean
   message?: string
+  scope?: string
+  workerLocation?: string
```

Comment:
Why are you adding them to `PrintStartMessageArgs`? 

You don't seem to be using `args.workerLocation` or `args.scope` anywhere.

You are only using `context` e.g. `context.worker?.scriptURL`.

Reply:
You're right I'm sorry I missed that. Thank you @kettanaito for fixing it.